pkg/latte/input_take_change_password.go

```diff
@@ -0,0 +1,50 @@
+package latte
+
+import (
+	"github.com/authgear/authgear-server/pkg/lib/workflow"
+	"github.com/authgear/authgear-server/pkg/util/validation"
+)
+
+func init() {
+	workflow.RegisterPublicInput(&InputTakeChangePassword{})
+}
+
+var InputTakeChangePasswordSchema = validation.NewSimpleSchema(`
+	{
+		"type": "object",
+		"additionalProperties": false,
+		"properties": {
+			"old_password": { "type": "string" },
+			"new_password": { "type": "string" }
+		},
+		"required": ["old_password", "new_password"]
+	}
+`)
+
+type InputTakeChangePassword struct {
```

Comment:
Maybe we can call the input `InputChangePassword`? `InputTakeChangePassword` has 2 verbs.